apps/docs/src/css/custom.css

```
@@ -246,32 +237,17 @@
-
-    50% {
-      background-position: 100% 100%;
-    }
 
-    to {
-      background-position: 0% 0%;
-    }
+  @keyframes shine {
+    0% { background-position: 0% 0%; }
+    50% { background-position: 100% 100%; }
+    to { background-position: 0% 0%; }
```

Stylelint (17.9.0)

[error] Expected "to" to be "100%" (keyframe-selector-notation)

(keyframe-selector-notation)